server/src/model/user.py

```diff
@@ -20,7 +20,7 @@ class User(Document, UserMixin):
     google_token = StringField()
     picture_url = StringField()
     email = StringField(required=True, max_length=320)
-    meta = {"collection": "users"}
+    meta = {"collection": "users", "strict": False}
```

Comment:
`strict:False` makes it so that if the data (in the db) has extra fields, a `FiledDoesNotExist` error does not occur. Instead the Document model will just ignore the extra filed and function as normal. 

This may happen during subtractive migration or during additive migration depending on the order of ops of deploy code and deploy migration. Can give examples if you want.